modules/examples/todolist/README.md

```diff
@@ -0,0 +1 @@
+# TodoList RPC example
```

Comment:
I'd add some basic instructions on how to run the application.

Reply:
Done!